src/objects.py

```diff
@@ -135,8 +135,8 @@ def __init__(self, accessor: PlanNodeAccessor, node_type):
         self.node_type: str = node_type
         self.level: int = 0
         self.name: str = None
-        self.properties = {}
-        self.child_nodes: List[PlanNode] = []
+        self.properties: Mapping[str: str] = dict()
```

Comment:
Will use:`from collections.abc import Iterable, Mapping` instead since the ones in `typing` are deprecated aliases.

>class typing.Iterable(Generic[T_co])
Deprecated alias to [collections.abc.Iterable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable).
Deprecated since version 3.9: [collections.abc.Iterable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).